qiling/loader/elf.py

```diff
@@ -615,4 +615,23 @@ def load_with_ld(self, stack_addr, load_address = -1, argv = [], env = {}):
         self.new_stack = new_stack
         self.load_address = load_address
         self.images.append(self.coverage_image(load_address, load_address + mem_end, self.path))
-        self.ql.os.function_hook = FunctionHook(self.ql, self.elf_phdr + mem_start, self.elf_phnum, self.elf_phent, load_address, load_address + mem_end)
\ No newline at end of file
+        self.ql.os.function_hook = FunctionHook(self.ql, self.elf_phdr + mem_start, self.elf_phnum, self.elf_phent, load_address, load_address + mem_end)
+
+        # map vsyscall section for some specific needs
+        _vsyscall_addr = 0xffffffffff600000
```

Comment:
this address is very "x8664 ?"

Reply:
you are right , its only for x8664 elf binary